analyzers/tests/SonarAnalyzer.Test/TestCases/SymbolicExecution/Roslyn/EmptyCollectionsShouldNotBeEnumerated.cs

```diff
@@ -647,7 +647,7 @@ public void LearnConditions_Size(bool condition, List<int> arg)
 
         if (empty.Count() == 0)
         {
-            empty.Clear();  // FN
+            empty.Clear();  // Noncompliant
```

Comment:
We got some freebies because of `.Count()` fix we came up with! (excluding `Enumerable` static extensions from being reset on `Argument.cs`)